tests/test_preprocessing.py

```diff
@@ -255,13 +255,13 @@ def check_scaling_features(use_feature_hashing=False, use_scaling=False):
 
     # these are the expected values of the f-measures, sorted
     if not use_feature_hashing:
-        expected_fmeasures = ([0.77319587628865982, 0.78640776699029125] if
+        expected_fmeasures = ([0.55276381909547745, 0.55721393034825872] if
                               not use_scaling else
-                              [0.94930875576036866, 0.93989071038251359])
+                              [0.65217391304347827, 0.70370370370370372])
```

Comment:
Yeah, looks like they [changed](http://scikit-learn.org/stable/whats_new.html#enhancements) the underlying function `sample_without_replacement()` which causes the samples to be different for the same random state. So, I think we can safely say that the difference in the results are because of that.